src/styles/utils/_animation.scss

```diff
@@ -0,0 +1,8 @@
+@keyframes move {
+  from {
+    transform: rotate(0);
+  }
+  to {
+    transform: rotate(360deg);
+  }
+}
```

Comment:
I have a smaller variant I've used, you may change it if you want 
`@keyframes rotate {
  100% {
    transform: rotateZ(360deg);
  }
}`